decidim-admin/app/commands/decidim/admin/create_participatory_process_admin.rb

```diff
@@ -35,19 +35,39 @@ def call
 
       attr_reader :form, :participatory_process
 
-      def create_participatory_process_admin
-        ParticipatoryProcessUserRole.create!(
-          role: :admin,
-          user: user,
-          participatory_process: @participatory_process
-        )
+      def create_role
+        if form.roles.include?("collaborator")
+          user.roles << "collaborator" unless user.roles.include?("collaborator")
```

Comment:
Then we need to redefine the whole "collaborator" role, we should open a new issue to do so. We need to define what abilities can a collaborator perform.

I'd go for working on this PR as it is, while we discuss the collaborator role. Then, if needed, open a new PR changing the collaborator abilities and the way to invite them.

What do you think?